modules/gapi/perf/common/gapi_video_perf_tests_inl.hpp

```diff
@@ -154,6 +154,79 @@ PERF_TEST_P_(BuildPyr_CalcOptFlow_PipelinePerfTest, TestPerformance)
 
 //------------------------------------------------------------------------------
 
+#ifdef HAVE_OPENCV_VIDEO
+
+PERF_TEST_P_(BackgroundSubtractorPerfTest, TestPerformance)
+{
+    namespace gvideo = cv::gapi::video;
+    initTestDataPath();
+
+    gvideo::BackgroundSubtractorType opType;
+    bool detectShadows = false;
+    double learningRate = -1.;
+    std::string filePath = "";
+    std::size_t testNumFrames = 0;
+    cv::GCompileArgs compileArgs;
+    CompareMats cmpF;
+
+    std::tie(opType, detectShadows, learningRate, filePath, testNumFrames,
+             compileArgs, cmpF) = GetParam();
+
+    int histLength = 500;
+    double thr = -1;
+    if (opType == gvideo::TYPE_BS_MOG2)
+        thr = 16.;
+    else if (opType == gvideo::TYPE_BS_KNN)
+        thr = 400.;
+    gvideo::BackgroundSubtractorParams bsp(opType, histLength, thr, detectShadows, learningRate);
+
+    // Video source declaration
+    cv::VideoCapture cap;
+    if (!cap.open(findDataFile(filePath))) throw SkipTestException("Video file can not be opened");
```

Comment:
Will it work on Centos? (Test skip)

Reply:
Yeah, I think it is all good. We have such a skip with `VideoCapture` in [infer_ie_tests](https://github.com/opencv/opencv/blob/master/modules/gapi/test/infer/gapi_infer_ie_test.cpp#L992)

Reply:
I hope this.